### PR TITLE
Introduce commit

### PR DESCRIPTION
**Pull request by gigbat**

### CODE REVIEW

**Review by AndreiYazadzhi — APPROVED** [review]

File: src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package com.dev.cinema.model.dto;
+
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.User;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java]: Nope

File: src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    private LocalDateTime orderDate;
+    private User user;
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java]: Nope

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/controller/UserController.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    private final UserMapper userMapper;
+
+    @Autowired
+    public UserController(UserService userService,
+                          UserMapper userMapper) {
+        this.userService = userService;
+        this.userMapper = userMapper;
+    }
+
+    @GetMapping
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/controller/UserController.java]: /users/by-email

File: src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java (hunk 1 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java]: done

File: src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java (hunk 2 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java]: done

File: src/main/java/com/dev/cinema/controller/UserController.java (hunk 3 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/controller/UserController.java]: done

File: src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java [hunk 4]
```diff
@@ -1,15 +1,12 @@
...
-import java.time.LocalDateTime;
 import java.util.List;
 
 public class OrderResponseDto {
     private Long id;
-    private List<Ticket> tickets;
-    private LocalDateTime orderDate;
-    private User user;
+    private List<Long> ticketsId;
+    private String orderDate;
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java]: could be LocalDateTime

**Review by boroda4436 — APPROVED** [review]

File: src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java (hunk 4 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java]: for request String, for a response can be Object, fixed

**Pull request merged**